### PR TITLE
Dis 17/comments crud

### PR DESCRIPTION
**Pull request by JSTheGreat**

### CODE REVIEW

File: src/main/java/com/dislinkt/post/controller/CommentController.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            ret.add(dto);
+        });
+        return ret;
+    }
+
+    @GetMapping()
+    public ResponseEntity<ResponseDTO<List<CommentDTO>>> getAll(){
+        List<CommentDTO> ret = service.findAll();
+        return new ResponseEntity<>(new ResponseDTO<>(ret), HttpStatus.OK);
+    }
```
Comment (mikecitt) [inline src/main/java/com/dislinkt/post/controller/CommentController.java:55]: Similiar to reactions. Do we need to query for all comments for all posts ever?

File: src/main/java/com/dislinkt/post/controller/CommentController.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            return new ResponseEntity<>(new ResponseDTO<>(ret), HttpStatus.OK);
+        }
+        catch (Exception ex){
+            ErrorDTO error = new ErrorDTO(ex.getMessage());
+            return new ResponseEntity<>(error, HttpStatus.BAD_REQUEST);
+        }
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity deletePost(@RequestHeader("X-User-Id") UUID personId, @PathVariable Integer id){
```
Comment (mikecitt) [inline src/main/java/com/dislinkt/post/controller/CommentController.java]:
```suggestion
    public ResponseEntity deleteComment(@RequestHeader("X-User-Id") UUID personId, @PathVariable Integer id){
```

**Review by mikecitt — DISMISSED** [review]
Nice! Just few nits

File: src/main/java/com/dislinkt/post/controller/CommentController.java (hunk 1 above)
Comment (JSTheGreat) [reply src/main/java/com/dislinkt/post/controller/CommentController.java:55]: Check for answer in reactions

**Review by mikecitt — DISMISSED** [review]

**Review by rodusek-v — APPROVED** [review]
All good!

**Pull request merged**